ansible/roles/pulp_site/defaults/main.yml

```diff
@@ -0,0 +1,43 @@
+pulp_site_url: "{{ appliances_pulp_url }}"
+pulp_site_port: 8080
+pulp_site_username: admin # shouldn't be changed
+pulp_site_upstream_content_url: https://ark.stackhpc.com/pulp/content
+pulp_site_upstream_username: slurm-app-ci
```

Comment:
Nope, there is no default for this really.

Reply:
CI username, will move to .stackhpc